### DISCUSSION TITLE
Air Gap upgrade to .182 fails on patch_os

### DISCUSSION THREAD

**Discussion by mtkhayes3** [Security-Onion-Solutions/securityonion#9402] [Unsupported Versions]
Currently running Air gap version .130 and trying to upgrade to .182 
It worked in test environment but I am only using 1 x Manager VM 1 x Sensor node and 1 x Search node 

Problem encountered upgrading packages. Additional info follows:
 result:
Running scope as a unit run-xxxx.scope.
Error: Package: libkadm5-1.15.1-54.el7_9.x86_64 (airgap_repo)
                 Requires: krb5-libs(x86_64) = 1.15.1-54.el7_9
                 Installed: krb5-libs-1.15.1.-51.el7_9.x86_64 (@airgap_repo)
                      krb5-libs(x86-64) = 1.15.1.-50.el7
Error: Package: krb5-devel-1.15.1-54.el7_9.x86_64 (airgap_repo)
                 Requires: krb5-libs(x86_64) = 1.15.1-54.el7_9
                 Installed: krb5-libs-1.15.1.-51.el7_9.x86_64 (@airgap_repo)
                      krb5-libs(x86-64) = 1.15.1.-50.el7
stdout: you could try using --skip-broken to work around the problem
            you could try running: rpm -Va --nofile --nodigest
Changes:
 ID: needs_restarting
Function: module.run
Result: True
Comment: min.send: True
Summary xxxxxx
Succeeded: 1 (changed=1)
Failed: 1
Command failed with exit code 2; will try in 60 seconds then fails altogeter

**Comment by dougburks** [comment]
Did you verify the .182 ISO image?

**Reply by mtkhayes3** [reply]
Yes it was and I also redownloaded it again just in case.  Should I try upgrading  to .160 first then .182?

**Reply by dougburks** [reply]
Sure, you could try that.

**Reply by mtkhayes3** [reply]
Ran libkadm5-1.15.1-54.el7_9.x86_64.rpm and Found that an rpm was missing from air gap repo krb-libs(x86-64) = 1.15.1.-54.el7_9 is missing even though I used the same iso for 3 other upgrades no issues and verified the iso

**Reply by dougburks** [reply]
Strange. Did you manually add any other software to that system or make any other changes?

**Comment by mtkhayes3** [comment]
I will try it and cross my fingers

**Comment by mtkhayes3** [comment]
So this is what I ended up doing and it is working now.
Instead of running U upgrade I ran C to continue without upgrades which
rebuilt the repo and then ran U upgrade.
I am not sure why the repo was missing some rpms but when I used the same
iso for other installs without any issues.
